charts/s1-agent/templates/helper/deployment.yaml

```diff
@@ -112,6 +112,8 @@ spec:
             mountPath: /s1-helper/agent-injection
     {{- if eq (include "serverlessOnlyMode" .) "true" }}
         {{- include "serverlessAgentContainer" . | nindent 8 }}
+          - name: S1_CONTAINER_NAME
```

Comment:
consider renaming to S1_AGENT_CONTAINER_NAME

Reply:
Reverted this change because @natishauli said customers depend on this.